docs/UserGuide.md

```diff
@@ -11,117 +11,213 @@
+Format6: add st/START_TIME et/ END_TIME
+
+>*Note: <br>
+ *DATE Format: DDMMYYYY*
+ *TIME Format: 0000 to 2359*
+
+Examples:
+1. `add sd/01102016`
+2. `add ed/11112016`
+3. `add sd/01102016 ed/11112016`
+4. `add st/0900`
+5. `add et/0900`
+6. `add st/0900 et/0900`
+
+####6.3 Listing all tasks: `list`
+Shows a list of all tasks in the MustDoList.<br>
 Format: `list`
 
-#### Finding all persons containing any keyword in their name: `find`
-Finds persons whose names contain any of the given keywords.<br>
-Format: `find KEYWORD [MORE_KEYWORDS]`
+####6.4 Finding all tasks containing any keyword in their task_name, date, time: `find`
+Finds task that contain any of the given keywords.<br>
+Format: `find KEYWORD`
 
-> The search is case sensitive, the order of the keywords does not matter, only the name is searched, 
-and persons matching at least one keyword will be returned (i.e. `OR` search).
+> The search is case sensitive, the order of the keywords does not matter (wait for feedback) and task matching at least one keyword will be returned (i.e. `OR` search).
 
 Examples: 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+* `find CS2103`<br>
+  Returns `Submission of CS2103 Project` but not `CS2103`
+* `find Submission of CS2103 Project`<br>
+  Returns Any task having names `Submission `, `CS2103 `, or `Project`
+* `find 0900`<br>
+  Returns Any task having `0900`
+
+####6.5 Select a task : `select`
+Selects the task identified by the index number used in the last task listing.<br>
+Format: `select INDEX`
 
-#### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
+> Selects the task at the specified `INDEX`. 
+  The index refers to the index number shown in the most recent listing.<br>
+  The index **must be a positive integer** 1, 2, 3, ...
+
+Examples: 
+* `list`<br>
+  `select 2`<br>
+  Selects the 2nd task in the MustDoList.
+* `find Submission` <br> 
+  `select 1`<br>
+  Selects the 1st task in the results of the `find` command.
+
+####6.6.1 Deleting a task: `delete`
+Deletes the specified task from the MustDoList. Irreversible.<br>
 Format: `delete INDEX`
 
-> Deletes the person at the specified `INDEX`. 
+> Deletes the task at the specified `INDEX`. 
   The index refers to the index number shown in the most recent listing.<br>
   The index **must be a positive integer** 1, 2, 3, ...
 
 Examples: 
 * `list`<br>
   `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
+  Deletes the 2nd task in the MustDoThis.
+* `find Submission`<br> 
   `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+  Deletes the 1st task in the results of the `find` command.
 
-#### Select a person : `select`
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
+####6.6.2 Deleting a task: `delete`
```

Comment:
Firstly, you need to change the title here, since it has the same title as above `delete` command. Secondly, I think it's better if you can append this feature into `edit` command.